src/Admin/Formation/PathAdmin.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace AppBundle\Admin\Formation;
+
+use Sonata\AdminBundle\Admin\AbstractAdmin;
+use Sonata\AdminBundle\Datagrid\ListMapper;
+use Sonata\AdminBundle\Form\FormMapper;
+use Symfony\Component\Form\Extension\Core\Type\TextareaType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+
+class PathAdmin extends AbstractAdmin
+{
+    protected function configureFormFields(FormMapper $formMapper)
+    {
+        $formMapper
+            ->with('Module', ['class' => 'col-md-12'])
+                ->add('title', TextType::class, [
+                    'label' => 'Titre',
+                ])
+                ->add('slug', TextType::class, [
+                    'label' => 'URL',
+                    'disabled' => true,
+                    'help' => 'Généré automatiquement depuis le titre.',
+                ])
+                ->add('description', TextareaType::class, [
+                    'label' => 'description',
+                ])
+            ->end()
+        ;
+    }
+
+    protected function configureListFields(ListMapper $listMapper)
+    {
+        $listMapper
+            ->addIdentifier('title', TextType::class, [
```

Comment:
you should not use `*Type::class` of form component for configuring list page.
you could keep it `null` and sonata will guess the type of displayed value or string `array`, '...'